cosmwasm/packages/wasmi-runtime/src/wasm/gas.rs

```diff
@@ -37,10 +37,10 @@ impl Default for WasmCosts {
             mem: 2,
             static_u256: 64,
             static_address: 40,
-            initial_mem: 4096,
+            initial_mem: 4096, // Assaf: I don't think this goes anywhere
             grow_mem: 8192,
             memcpy: 1,
-            max_stack_height: 64 * 1024,
+            max_stack_height: 64 * 1024, // Assaf: I don't think this goes anywhere
```

Comment:
what do you mean?

Reply:
By looking at how the gas metering works, this isn't used.

Reply:
Look at https://github.com/enigmampc/SecretNetwork/blob/1ff31db4940e5162c71a2691f98aac60a753336f/cosmwasm/packages/wasmi-runtime/src/wasm/gas.rs#L50-L72

Reply:
huh, that's weird...
I think we should investigate what this does - maybe we're using it wrong - but that's a separate issue.

Reply:
Oh I got it. I think `initial_mem` was being used as `regular`. Fixed it in 86d1ae1e8cc7b9e4080d66073fe95794b0e6814f

Reply:
ah, awesome! thanks for taking care of it!

Reply:
Asked parity about controlling the stack usage: https://matrix.to/#/!OoAWoTkXgxMLnLwGPO:matrix.parity.io/$1596011601124301ImAZX:matrix.org?via=matrix.parity.io&via=matrix.org